MetaMask/metamask-extension
# fix: Buy button not disabled for ERC20 tokens in Sepolia

## Explanation

Fixes #18949 

**Problem**: whenever we import tokens on Sepolia network, wee can see the Buy button is not disabled.
**Expected**: Buy button should be disabled as we do on Goerli or any other testnets.

**Solution** : 
As we can see on `shared/constants/network.ts` at  `BUYABLE_CHAINS_MAP` Testnets are not buyable chains while only selected mainnets are buyable.

I just shifted Sepolia to non buyable chains likewise other testnets




## Screenshots/Screencaps



### Before


<img width="1438" alt="Sepolia 2" src="https://github.com/MetaMask/metamask-extension/assets/32412967/57b8a37b-31c9-4e5d-93fe-c65dd9937a8c">

### After


<img width="1439" alt="Sepolia 2" src="https://github.com/MetaMask/metamask-extension/assets/32412967/cd1fe15d-bf67-45de-aed0-b95758940166">


## Manual Testing Steps
 

1. Select Sepolia network
2. Go to test dapp https://metamask.github.io/test-dapp/
3. Click Create Token and procced
4. Click Add Token to wallet and proceed
5. Go to MetaMask
6. Click the new added token
7. See Buy button is disabled

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
